components/raftstore/src/store/fsm/peer.rs

```diff
@@ -619,7 +620,12 @@ where
     pub fn handle_msgs(&mut self, msgs: &mut Vec<PeerMsg<EK>>) {
         let timer = TiInstant::now_coarse();
         let count = msgs.len();
+        let mut distribution = hash_map_with_capacity(std::mem::variant_count::<PeerMsg<EK>>());
```

Comment:
How about putting it to `PollContext`? So that we can save frequent hashmap creation.
Also it could be an fixed size array as `variant_count::<PeerMsg>` is a constant number.

Reply:
Using a fixed size array instead, as it's should be allocated on stack, I don't put it in `PollContext`